### DISCUSSION TITLE
Tip for anyone wanting to run FreeTube nightly builds on Windows 7

### DISCUSSION THREAD

**Discussion by KandiPanda** [FreeTubeApp/FreeTube#4476] [General]
Tip for running FreeTube nightly builds on Windows 7, since otherwise _any_ nightly after Release 0.19.1 Beta seems to fail in various ways.

**Edit: This is because Windows 7, 8, and 8.1 are no longer supported.**

Tested working using [freetube-0.19.1-nightly-3746](https://github.com/FreeTubeApp/FreeTube/actions/runs/7278585188) (selected simply because it was the latest nightly build at the moment), but _should_ work for releases back to [freetube-0.19.1-nightly-3736](https://github.com/FreeTubeApp/FreeTube/actions/runs/7201254253) ("Bump electron from 27.1.3 to 28.0.0"). (Edit: Tested, and 3736 does also work.)

Does _not_ seem to work (tested previously) for older releases such as [freetube-0.19.1-nightly-3546](https://github.com/FreeTubeApp/FreeTube/actions/runs/6592954983) ("Upgrade electron from 22.x to 27.x and replace deprecated `protocol.r…") or [freetube-0.19.1-nightly-3635](https://github.com/FreeTubeApp/FreeTube/actions/runs/6863322210) (probably just the latest nightly build the last time I tested it).

Start with https://github.com/win32ss/supermium-electron (I used the 64-bit version of [this specific release](https://github.com/win32ss/supermium-electron/releases/tag/v28-testing), currently the *only* release) and `freetube-0.19.1-nightly-xxxx-win-x64-portable.zip` from the build page artifacts (with _xxxx_ being the particular nightly build that you want to use).

- Extract the contents of that into the folder you want it to be "installed" in.
- Delete the contents of its resources folder (per their instructions).
- Open `freetube-0.19.1-nightly-xxxx-win-x64-portable.zip`.
- Copy the contents of its resources folder into Supermium-Electron's resources folder (per their instructions).
- Run electron.exe (per their instructions).

Do note that it will have the generic Electron icon in the title bar and taskbar rather than the FreeTube icon, though I imagine that that's fixable if someone cares enough (either through using a resource editor on the exe, or by building it yourself from source using the icon you want in place of `supermium-electron/default_app/icon.png`).

Also, I did _not_ test with the 32-bit version of Supermium-Electron (I have no need for that), but if anyone tries it and it works, feel free to comment so that other people who for whatever reason do need that will know one way or the other if it works.

(Also note that while I don't actually know, unless told otherwise, I will be assuming that this is an _unsupported_ configuration and that I'm on my own with any problems that come up with it.)

**Comment by absidue** [comment]
Glad you got it working, but yes that is a very much unsupported configuration. We can't promise that you will have a usable experience in the future. Chromium dropped support for Windows 7, 8 and 8.1, so Electron had to as well. FreeTube, which uses the official Electron releases, will do too (it'll be in the next release notes, but as you are using the nightly builds you have discovered it early).

If your machine is too old to run newer versions of Windows, I recommend switching to Linux as that is more lightweight and will let you use up-to-date stuff.

**Reply by KandiPanda** [reply]
Yeah, I'll probably stick to the 0.19.1 Beta release version for now unless YouTube breaks something or there's some big change that I really want. But I figured that other people might find the information useful. (I think someone was even asking about 32-bit a while back? While I didn't try it with the 32-bit Supermium-Electron, if that works then it might help those people as well.)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Archiving this.
